cmd/hz/internal/protobuf/tags.go

```diff
@@ -51,6 +52,9 @@ func CheckTagOption(args *config.Argument) (ret []generator.Option) {
 	if args.JSONEnumStr {
 		ret = append(ret, generator.OptionMarshalEnumToText)
 	}
+	if args.PbJsonTag {
```

Comment:
Json -> JSON? naming consistency

Reply:
Most of the code about "json" is named "json" or "Json". Here we replace "JSON" with "Json"